test/config/tests.bootstrap.js

```diff
@@ -9,4 +9,5 @@ global.sinon = require('sinon')
 require('sinon-mongoose')
 require('sinon-as-promised')
 
-global.config = require('../../src/config/config')()
+global.config = require('../../src/config/config')
+global.clearModule = require('clear-module')
```

Comment:
when you use `require`, the module gets cached in memory. That's just how node works. We're using `clear-module` in order to reset the module so we could test it. **It's worth mentioning that it's being used just for testing purposes**.